### PR TITLE
Update azure-stack-vaas-set-up-resources.md

### PR DESCRIPTION
**Pull request by JohnMarlin-MSFT**
Added best practice recommendation in "Assign User Roles" section.

### CODE REVIEW

**Comment by PRMerger6** [comment]
@JohnMarlin-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/partner/azure-stack-vaas-set-up-resources.md [hunk 1]
```diff
@@ -96,6 +96,8 @@ To assign roles for the VaaS service app:
 4. Select **Users and groups**. The **AzureStack Validation Management Api [wsfed enabled] [Migrated] - Users and group** blade lists the users with permission to use the app.
 5. Select **+ Add user** to add a user from your tenant and assign a role.
 
+NOTE: It is a best practice and strong recomendation that more than one Owner is defined.  A new tenant must be ceated if there is a problem, departure, etc with a single Owner.
```
Comment (sethmanheim) [inline azure-stack/partner/azure-stack-vaas-set-up-resources.md:99]:
```suggestion
> [!NOTE]
> It is a best practice and strong recommendation that more than one owner is defined.  A new tenant must be created if there is a problem or departure with a single owner.
```

Comment (sethmanheim) [reply azure-stack/partner/azure-stack-vaas-set-up-resources.md:99]: Please use correct Markdown formatting and check spelling before submitting.

**Comment by jborsecnik** [comment]
@sethmanheim, if you want, the PR review team can check/correct format/spelling and merge this PR. Let us know, and add a `#sign-off` comment. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by sethmanheim** [comment]
@jborsecnik That's be great, thank you. #sign-off

**Comment by PRMerger19** [comment]
@Jak-MS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**